base/gmtime.hpp

```diff
@@ -5,5 +5,5 @@
 namespace base
 {
 /// A cross-platform replacenemt of gmtime_r
-std::tm GmTime(time_t const time);
+std::tm GmTime(time_t time);
```

Comment:
Correspondent cpp change?

Reply:
Clang-Tidy: Parameter 'time' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions